core-build/gulp-core-build-sass/src/SassTask.ts

```diff
@@ -78,6 +78,11 @@ export interface ISassTaskConfig {
    * returnPromise and sourceMap will be ignored.
    */
   cleanCssOptions?: CleanCss.Options;
+
+  /**
+   * Allows the override of the options passed to autoprefixer.
+   */
+  autoprefixerOptions?: autoprefixer.Options;
```

Comment:
Can you pass what was already there as the defaults in the constructor?

Reply:
Done in https://github.com/microsoft/rushstack/pull/1891/commits/07623c99ff9297c9d449d9cee3e886347f655aeb.